modules/aws-outposts-load-balancer-clb-fix-subnet.adoc

```diff
@@ -0,0 +1,121 @@
+
+You can prevent Classic Load Balancers from being added to the Outpost availability zone by adding an annotation with the correct subnet.
+
+[NOTE]
+====
+The steps to ensure that Classic Load Balancers use the correct subnet are a workaround for clusters installed with {product-title} {product-version}.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
+
+.Procedure
+
+. #How does the user find these to edit them? Or is this assuming from new?#
+
+. Edit the Classic Load Balancer `deployment` resource:
++
+.Example deployment configuration
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <application_name>
+  namespace: <application_namespace>
+spec:
+  selector:
+    matchLabels:
+      app: <application_name>
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: <application_name>
+    spec:
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: <application_name>
+          ports:
+            - containerPort: 8080
+----
+
+. Create the `deployment` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_deployment>.yaml
+----
+
+. Edit the Classic Load Balancer `service` resource:
++
+.Example service configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: <application_name>
+  name: <application_name>
+  namespace: <application_namespace>
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-subnets: <aws_subnet> # <1>
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: <application_name>
+  type: LoadBalancer
+----
+<1> Specify the cloud-based subnet.
+
+. Create the `service` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_service>.yaml
+----
+
+.Verification
+
+* View the Classic Load Balancer status by running the following command:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,text]
+----
+NAME                 TYPE           CLUSTER-IP              EXTERNAL-IP              PORT(S)        AGE
+<application_name>   LoadBalancer   <cluster_ip_address>    <external_ip_address>    80:32199/TCP   38m
+----
++
+#What are they looking for in this to confirm it's good?#
```

Comment:
we can refer to the last step `Verification`[here](https://docs.openshift.com/container-platform/4.14/networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.html) , we can use
```
$ HOST=$(oc get svc <application_name> -n <application_namespace> --template='{{(index .status.loadBalancer.ingress 0).hostname}}')
$ curl $HOST
```